package.json

```diff
@@ -45,17 +50,23 @@
     "brace": "^0.8.0",
     "browserify": "^13.0.0",
     "csslint": "^1.0.2",
+    "env-cmd": "github:serapath-contribution/env-cmd",
```

Comment:
I am not so fan of referencing a github module that is outside of the Ethereum scope.
Is this a bug fix?